internal/tasks/device_disconnected.go

```diff
@@ -36,11 +36,11 @@ func (t *DeviceDisconnected) Poll(ctx context.Context) {
 	defer cancel()
 
 	// Calculate the cutoff time for disconnected devices
-	cutoffTime := time.Now().Add(-api.DeviceDisconnectedTimeout)
+	cutoffTime := time.Now().Add(-api.DeviceDisconnectedTimeout).Format(time.RFC3339)
```

Comment:
just for my own understanding why these changes.

Reply:
I can revert.